packages/edit-site/src/components/global-styles/font-library-modal/upload-fonts.js

```diff
@@ -94,6 +115,23 @@ function UploadFonts() {
 		handleInstall( fontFacesLoaded );
 	};
 
+	/**
+	 * Checks if a file is a valid Font file.
+	 *
+	 * @param {File} file The file to be checked.
```

Comment:
```suggestion
	 * @param {File} The file to be checked.
```

Reply:
`file` is the parameter name and required. It's unclear in the diff but will be clearer in IDEs.

![Screen Shot 2024-03-08 at 9 26 33 am](https://github.com/WordPress/gutenberg/assets/519727/fc143d8a-d8ae-45d9-886e-68d4c926b7f1)